Codeinwp/otter-blocks
# Dynamic Value Additions

Closes #1218.


### Summary

- Added Parameter based Dynamic Value
- Added a country location for Dynamic Value & Conditions using https://www.geoplugin.com
- Add Cookie based Block Conditions

### Screenshots 

----

### Test instructions


1. A parameter-based dynamic value, so you can show a value, let's say brand, based on if the user has that keyword as a parameter in their URL, like example.com?brand=puma, then it will display Puma.

2. A location-based Dynamic Condition to show your country and Block conditions. It's based on your IP so might not be correct 100% of the times, maybe test with VPN too to see.

3. It's similar to how Meta works but just with PHP cookies. You can see how you can add cookie to your website, you can use the otter-blocks.php file to add the code and view the website in the frontend: https://www.w3schools.com/php/php_cookies.asp

---- 

### Checklist before the final review
